src/transformers/configuration_utils.py

```diff
@@ -136,7 +136,7 @@ class PretrainedConfig(PushToHubMixin):
         diversity_penalty (`float`, *optional*, defaults to 0.0):
             Value to control diversity for group beam search. that will be used by default in the `generate` method of
             the model. 0 means no diversity penalty. The higher the penalty, the more diverse are the outputs.
-        temperature (`float`, *optional*, defaults to 1):
+        temperature (`float`, *optional*, defaults to 1.0):
```

Comment:
The type is clearly indicated, not sure what this change brings honestly.

Reply:
Ah, I just saw that other ones with `float` as its type (like `diversity_penalty (float, *optional*, defaults to 0.0)` right above) had its value formatted like `0.0`, so I thought I'd just change it for consistency